lsp-mode.el

```diff
@@ -6324,6 +6324,8 @@ execute a CODE-ACTION-KIND action."
 
 (lsp-make-interactive-code-action organize-imports "source.organizeImports")
 
+(lsp-make-interactive-code-action remove-unused-imports "source.removeUnusedImports")
```

Comment:
I dont see that source on https://microsoft.github.io/language-server-protocol/specifications/lsp/3.18/specification/#codeActionKind

Reply:
Uh, well... I assumed it was in the spec because typescript language server supports it: https://github.com/typescript-language-server/typescript-language-server?tab=readme-ov-file#features

How do we proceed? I mean, it is easy enough to make the function call I added in the PR outside of the module, but I thought, it would be helpful for others if this was part of lsp-mode...

Reply:
If only a single language server is using, not sure it's a good idea add a function to whole lsp-mode, probably something specific for ts is best, in the lsp-typescript only.

Reply:
Got it, did some changes. Does this work for you?